src/main/java/makePost/use_case/MakePostException.java

```diff
@@ -0,0 +1,14 @@
+package makePost.use_case;
```

Comment:
It might be better to put this class in a separate folder (e.g. inside `src/main/java/exceptions`).